qa-tests-backend/src/main/groovy/services/NetworkGraphService.groovy

```diff
@@ -41,6 +41,20 @@ class NetworkGraphService extends BaseService {
         }
     }
 
+    static getExternalNetworkEntities(String query = null) {
+        try {
+            GetExternalNetworkEntitiesRequest.Builder request =
+                   GetExternalNetworkEntitiesRequest.newBuilder()
+                           .setClusterId(ClusterService.getClusterId())
+            if (query != null) {
+                request.setQuery(query)
+            }
+            return getNetworkGraphClient().getExternalNetworkEntities(request.build())
+        } catch (Exception e) {
+            log.error("Exception fetching external entities", e)
```

Comment:
I think it's better to throw the exception than return null

Reply:
This has now been removed.